### ISSUE TITLE
I have this Error when I want to create a new project with Angular 

### ISSUE THREAD

**Issue by Hicham95** [JeremyEnglert/JointsWP#386]
npm WARN deprecated circular-json@0.5.9: CircularJSON is in maintenance only, flatted is its successor.

> node-sass@4.9.3 install D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\node-sass
> node scripts/install.js

Downloading binary from https://github.com/sass/node-sass/releases/download/v4.9.3/win32-x64-67_binding.node
Cannot download "https://github.com/sass/node-sass/releases/download/v4.9.3/win32-x64-67_binding.node":

HTTP error 404 Not Found

Hint: If github.com is not accessible in your location
      try setting a proxy via HTTP_PROXY, e.g.

      export HTTP_PROXY=http://example.com:1234

or configure npm proxy via

      npm config set proxy http://example.com:8080

> node-sass@4.9.3 postinstall D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\node-sass
> node scripts/build.js

Building: C:\Program Files\nodejs\node.exe D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\node-gyp\bin\node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=
gyp info it worked if it ends with ok
gyp verb cli [ 'C:\\Program Files\\nodejs\\node.exe',
gyp verb cli   'D:\\Web Developement Tutorials\\Angular\\CheckListApp\\node_modules\\node-gyp\\bin\\node-gyp.js',
gyp verb cli   'rebuild',
gyp verb cli   '--verbose',
gyp verb cli   '--libsass_ext=',
gyp verb cli   '--libsass_cflags=',
gyp verb cli   '--libsass_ldflags=',
gyp verb cli   '--libsass_library=' ]
gyp info using node-gyp@3.8.0
gyp info using node@11.2.0 | win32 | x64
gyp verb command rebuild []
gyp verb command clean []
gyp verb clean removing "build" directory
gyp verb command configure []
gyp verb check python checking for Python executable "python2" in the PATH
gyp verb `which` failed Error: not found: python2
gyp verb `which` failed     at getNotFoundError (D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\which\which.js:13:12)
gyp verb `which` failed     at F (D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\which\which.js:68:19)
gyp verb `which` failed     at E (D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\which\which.js:80:29)
gyp verb `which` failed     at D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\which\which.js:89:16
gyp verb `which` failed     at D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\isexe\index.js:42:5
gyp verb `which` failed     at D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\isexe\windows.js:36:5
gyp verb `which` failed     at FSReqCallback.oncomplete (fs.js:161:21)
gyp verb `which` failed  python2 { Error: not found: python2
gyp verb `which` failed     at getNotFoundError (D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\which\which.js:13:12)
gyp verb `which` failed     at F (D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\which\which.js:68:19)
gyp verb `which` failed     at E (D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\which\which.js:80:29)
gyp verb `which` failed     at D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\which\which.js:89:16
gyp verb `which` failed     at D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\isexe\index.js:42:5
gyp verb `which` failed     at D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\isexe\windows.js:36:5
gyp verb `which` failed     at FSReqCallback.oncomplete (fs.js:161:21)
gyp verb `which` failed   stack:
gyp verb `which` failed    'Error: not found: python2\n    at getNotFoundError (D:\\Web Developement Tutorials\\Angular\\CheckListApp\\node_modules\\which\\which.js:13:12)\n    at F (D:\\Web Developement Tutorials\\Angular\\CheckListApp\\node_modules\\which\\which.js:68:19)\n    at E (D:\\Web Developement Tutorials\\Angular\\CheckListApp\\node_modules\\which\\which.js:80:29)\n    at D:\\Web Developement Tutorials\\Angular\\CheckListApp\\node_modules\\which\\which.js:89:16\n    at D:\\Web Developement Tutorials\\Angular\\CheckListApp\\node_modules\\isexe\\index.js:42:5\n    at D:\\Web Developement Tutorials\\Angular\\CheckListApp\\node_modules\\isexe\\windows.js:36:5\n    at FSReqCallback.oncomplete (fs.js:161:21)',
gyp verb `which` failed   code: 'ENOENT' }
gyp verb check python checking for Python executable "python" in the PATH
gyp verb `which` succeeded python C:\Users\Hicham\AppData\Local\Programs\Python\Python37-32\python.EXE
gyp ERR! configure error
gyp ERR! stack Error: Command failed: C:\Users\Hicham\AppData\Local\Programs\Python\Python37-32\python.EXE -c import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack   File "<string>", line 1
gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack                                ^
gyp ERR! stack SyntaxError: invalid syntax
gyp ERR! stack
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:294:12)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at maybeClose (internal/child_process.js:978:16)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:265:5)
gyp ERR! System Windows_NT 10.0.10240
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "D:\\Web Developement Tutorials\\Angular\\CheckListApp\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd D:\Web Developement Tutorials\Angular\CheckListApp\node_modules\node-sass
gyp ERR! node -v v11.2.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
Build failed with error code: 1
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: node-sass@4.9.3 (node_modules\node-sass):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: node-sass@4.9.3 postinstall: `node scripts/build.js`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1

**Comment by mlangone** [comment]
I believe this is a version number error for node sass please check that
your package.jspm sass version is the same as the one on the repo. There
was a new gulpfile and package.jsono  update I would recommend grabbing
them from the repo and replacing yours, then rm -rf node_modules, then npm
install again. Please backup first.

ML

**Comment by garrett-eclipse** [comment]
Hello @Hicham95, I'm going to close this as your issues are unrelated to JWP itself and are more due to customizations for Angular and your specific project. That being said will still try and give you a hand here.

First issue... CircularJSON is deprecated;
`npm WARN deprecated circular-json@0.5.9: CircularJSON is in maintenance only, flatted is its successor.`
CircularJSON - https://www.npmjs.com/package/circular-json
flatted - https://github.com/WebReflection/flatted#flatted
This is just a warning so can be ignored, but if it's a long-standing project you'll want to update the dependancy to flatted which may require customizations.

Second issue... failure to download binary;
```
Cannot download "https://github.com/sass/node-sass/releases/download/v4.9.3/win32-x64-67_binding.node":
HTTP error 404 Not Found
```
Taking a look it's a broken URL so you'll need to determine where it's set and update to a working URL, either of these work;
Latest - https://github.com/sass/node-sass/releases/download/v4.11.0/win32-x64-67_binding.node
Same Version (4.9.3) - https://github.com/sass/node-sass/releases/download/v4.9.3/win32-x64-64_binding.node
Comparing to the broken URL the issue is the broken and latest use `x64-67` but in version 4.9.3 it was `x64-64`. If you can't just fix the URL maybe just update to use v4.11.0

Finally there's mention of python not found which as @mlangone mentioned may be node sass version issue. There's two threads that might fix this or have advice for python on windows;
https://github.com/JeremyEnglert/JointsWP/issues/317
https://github.com/JeremyEnglert/JointsWP/issues/352

Best of luck, and if you have any more issues you can ping back on this issue and I'll take a look.

Cheers
